system-tests/jobs/test-jobs-cy.js

```diff
@@ -1,185 +1,47 @@
-    cy.root()
-      .getFormGroupInputFor("Job ID *")
-      .should("have.value", `${fullJobName}`);
-
-    cy.root().getFormGroupInputFor("Mem (MiB) *").should("have.value", "32");
-
-    cy.root().get("label").contains("Command Only").click();
-
-    cy.root()
-      .getFormGroupInputFor("Command *")
-      .should("have.value", `${cmdline}`);
-  });
-
-  it("creates a job with default ucr config", () => {
     const jobName = "ucr";
-    const fullJobName = `${Cypress.env("TEST_UUID")}.${jobName}`;
     const cmdline = "while true; do echo 'test' ; sleep 100 ; done";
 
-    // Visit jobs
-    cy.visitUrl("jobs/overview");
-
-    // Click 'Create a job'
-    // Note: The current group contains the previous job
-    cy.get(".button.button-primary-link.button-narrow").click();
-
-    // Wait for the 'New Job' dialog to appear
-    cy.get(".modal-header").contains("New Job");
-
-    // Fill-in the input elements
-    cy.root()
-      .getFormGroupInputFor("Job ID *")
-      .type(`{selectall}${fullJobName}`);
-
-    cy.root().getFormGroupInputFor("CPUs *").type("{selectall}0.5");
-
-    cy.root().getFormGroupInputFor("Mem (MiB) *").type("{selectall}32");
-
-    // Select `Container Image` radio button
-    cy.root().get("label").contains("Container Image").click();
-    cy.root().getFormGroupInputFor("Command").type(cmdline);
-
-    // Fill-in image
-    cy.root().getFormGroupInputFor("Container Image *").type("nginx");
-
-    // Click crate job
-    cy.contains("Submit").click();
-
-    // Switch to the group that will contain the service
-    cy.visitUrl(`jobs/overview/${Cypress.env("TEST_UUID")}`);
-
-    // Wait for the table and the service to appear
-    cy.get(".page-body-content table")
-      .contains(jobName)
-      .get("a.table-cell-link-primary")
-      .contains(`${jobName}`)
-      .click();
-
-    // open edit screen
-    cy.get(".page-header-actions .dropdown")
-      .click()
-      .get(".dropdown-menu-items")
-      .contains("Edit")
-      .click();
-
-    // Fill-in the input elements
-    cy.root()
-      .getFormGroupInputFor("Job ID *")
-      .should("have.value", `${fullJobName}`);
-
-    cy.root().getFormGroupInputFor("CPUs *").type("{selectall}0.5");
-    cy.root().getFormGroupInputFor("Mem (MiB) *").should("have.value", "32");
-    cy.root()
-      .getFormGroupInputFor("Command")
-      .should("have.value", `${cmdline}`);
-
-    // Fill-in image
-    cy.root()
-      .getFormGroupInputFor("Container Image *")
-      .should("have.value", "nginx");
-  });
-
-  it("runs, stops and deletes a job", () => {
-    // first create a simple job
-    const jobName = "delete";
-    const fullJobName = `${Cypress.env("TEST_UUID")}.${jobName}`;
-    const cmdline = "while true; do echo 'test' ; sleep 100 ; done";
-
-    // Visit jobs
-    cy.visitUrl("jobs/overview");
-    cy.get(".button.button-primary-link.button-narrow").click();
+    // ////////////////////////////////////////////////////////////////////////
```

Comment:
Is this the proper way of organizing tests? Why not use `it` or `context`?

Reply:
those are system-tests against a real cluster.

team formerly wrote those tests like unit tests, often times repeating the process or creating and deleting a service lots of times just to test e.g. whether different form fields persist their value.

having many assertions in a single run, speeds up our tests massively and also proved to reduce flakiness for integration- and system-tests.

while one could argue that we could put `it`s inbetween, i think that an `it` should be self-contained, so that one can move it freely in the according file.